frontend/src/components/widgets/inputs/FontInput.vue

```diff
@@ -112,34 +112,45 @@ export default defineComponent({
 			if (!this.disabled) (this.$refs.menuList as typeof MenuList).setOpen();
 		},
 		selectFont(newName: string) {
-			if (this.isStyle) this.$emit("update:fontStyle", newName);
-			else this.$emit("update:fontFamily", newName);
-
-			{
-				const fontFamily = this.isStyle ? this.fontFamily : newName;
-				const fontStyle = this.isStyle ? newName : getFontStyles(newName)[0];
-				const fontFile = getFontFile(fontFamily, fontStyle);
-				this.$emit("changeFont", { fontFamily, fontStyle, fontFile });
```

Comment:
You decided to remove the ternary operator here? I felt it was much more readable before. Is this just a stylistic choice or am I missing something?

Reply:
It checked the case three times, inconsistently mixed between an `if/else` pair and two ternaries checking the same case. It felt very strange to segregate the three checks into different styles and basically interlace the checks based on the same condition. Extending the existing `if/else` block so it more clearly segregates the two possible situations and applies all the control flow for each branch in order means that the intent is less obfuscated. It also prevents bugs arising from accidentally toying with the two ternaries or the one-liner `if/else` blocks individually causing inconsistent behavior. We want to do the check exactly once and branch into the outcomes from the situation, rather than branching three times for the three outcomes.

Reply:
Fair enough. I still stand by it's readability having gone down, but I can agree with your assessment and reasoning.